modules/37_industry/subsectors/datainput.gms

```diff
@@ -604,18 +604,23 @@ $ifthen.cm_subsec_model_steel "%cm_subsec_model_steel%" == "processes"
 !! carbon capture mass is given in tCO2 and converted to tC after that
 
 !! reduction: 504 m^3; heat 242 m^3; conversion: x / 11.126 m^3/kg * 0.0333 MWh/kg
-p37_specFeDemTarget("feh2s","idr","h2")           = 2.23 / (sm_TWa_2_MWh/sm_giga_2_non);    !! Source: POSTED / Rechberger et al 2020, Section 4.2 (per tDRI)
```

Comment:
Could you please also update the comment in the line above, i.e. how you changed the calculation?

Where did you find the differing value?